Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,74 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+nav{
+    background-color: white;
+}
+
+.container{
+    position: relative;
+    background-color: black;
+}
+
+.flowershop{
+    width: 100%;
+    height: auto;
+    opacity: 0.5;
+    
+}
+
+h1{
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+    vertical-align: center;
+    color: antiquewhite;
+    position: absolute;
+    top: 30%;
+    width: 100%;
+    font-size: 50px;
+}
+
+button{
+    padding: 8px;
+    position: absolute;
+    top: 60%;
+    left: 40%;
+    width: 20%;
+    background-color: green;
+    font-size: 20px;
+    color: white;
+    border-radius: 8px;
+    border-style: none;
+    
+}
+
+button:hover{
+    background-color: lightgreen;
+
+}
+
+button:active{
+    background-color: darkgreen;
+}
+
+.container2{
+    position: relative;
+    background-color: lightgreen;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between; 
+    padding: 10% 15%;
+}
+
+.content{
+    width: 500px;
+    padding: 50px auto;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 2.5rem;
+}
+
+.symbol{
+    float: left;
+    padding-right: 20px;
+}
```

Comment:
You've recreated the website in a beautiful way. Well done!

Reply:
> You've recreated the website in a beautiful way. Well done!

Thanks for all the inputs, Bibek